testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -34,11 +37,14 @@
 import itertools
 
 from MDAnalysisTests.plugins.knownfailure import knownfailure
+from MDAnalysisTests import parser_not_found
 
 
 class TestAtom(TestCase):
     """Tests of Atom."""
 
+    @dec.skipif(parser_not_found('DCD'),
```

Comment:
isn't it ok to let all this stuff fail. We tell travis that a failed python3 test is ok for the moment.